src/sentry/quotas/redis.py

```diff
@@ -96,7 +129,7 @@ def get_usage_for_quota(client, quota):
                 return (None, None)
 
             key = self.__get_redis_key(
-                quota.key, timestamp, quota.window, organization_id % quota.window
```

Comment:
Might as well push the `shift` expression down into `__get_redis_key` if all of the local variables necessary to calculate it are going to be there anyway.

Reply:
True, although `shift` is used outside of that function in many cases, too, so I'd opt to leave it in this case.